.github/workflows/ci.yml

```diff
@@ -11,6 +11,27 @@ env:
   MYPY_FORCE_TERMINAL_WIDTH: "180"
 
 jobs:
+  build:
```

Comment:
nit: is this the clearest name for the build job?

![image](https://github.com/qutebrowser/qutebrowser/assets/7419144/a37708b4-ae3f-4990-b537-e6f31bb088b4)

Like to me it kinda implies build artifact and ... do something with it? Also what does "build" mean here? There is "package" used elsewhere which I think does help clarify it a bit.

Reply:
> Like to me it kinda implies build artifact and ... do something with it? 

I had a similar feeling.

> Also what does "build" mean here?

"build" means "build package" here. So I agree "package" would be a better fit.